bin/p2-bootstrap/bootstrap.go

```diff
@@ -39,7 +40,7 @@ func main() {
 		if err != nil {
 			log.Fatalf("Could not get consul manifest: %s", err)
 		}
-		consulPod = pods.NewPod(consulManifest.ID(), pods.PodPath(*podRoot, consulManifest.ID()))
+		consulPod = pods.NewPod(string(consulManifest.ID()), pods.PodPath(*podRoot, consulManifest.ID()))
```

Comment:
The call sites for NewPod are pretty evenly split between callers wanting to pass a string vs an id pulled from some other manifest. I felt like it's philosophically better to make NewPod do the cast for you, but if you feel strongly I can change it